internal/testing/e2e/backfill_test.go

```diff
@@ -132,20 +131,39 @@ func TestBackfillFrontendLifecycle(t *testing.T) {
 	require.Nil(t, get)
 }
 
-func TestProposedBackfillCreate(t *testing.T) {
-	ctx := context.Background()
+func TestAcknowledgeBackfill(t *testing.T) {
 	om := newOM(t)
+	ctx := context.Background()
 
-	t1, err := om.Frontend().CreateTicket(ctx, &pb.CreateTicketRequest{
-		Ticket: &pb.Ticket{
-			SearchFields: &pb.SearchFields{
-				StringArgs: map[string]string{
-					"field": "value",
-				},
-			},
+	bf := &pb.Backfill{SearchFields: &pb.SearchFields{
+		StringArgs: map[string]string{
+			"search": "me",
 		},
-	})
-	require.Nil(t, err)
+	},
+	}
+	createdBf, err := om.Frontend().CreateBackfill(ctx, &pb.CreateBackfillRequest{Backfill: bf})
+	require.NoError(t, err)
+	require.Equal(t, int64(1), createdBf.Generation)
+
+	ticketID := createMatchWithBackfill(ctx, om, createdBf, t)
+	conn := "127.0.0.1:4242"
+	getBF, err := om.Frontend().AcknowledgeBackfill(ctx, &pb.AcknowledgeBackfillRequest{BackfillId: createdBf.Id, Assignment: &pb.Assignment{Connection: conn, Extensions: map[string]*any.Any{
```

Comment:
tests with parameters validation are missing. seems you are testing only a happy path.

Reply:
Well I think parameters validation can form a new test - `frontend_service_test.go`